src/patches/slice_stats.cpp

```diff
@@ -0,0 +1,58 @@
+#include <lsqecc/patches/slice_stats.hpp>
+
+namespace lsqecc {
+
+std::ostream& operator<<(std::ostream& os, const SliceStats& slice_stats)
+{
+    auto get_percentage_volume = [&](size_t x) -> double {
+                    return static_cast<double>(x) / static_cast<double>(slice_stats.totals.volume) * 100.0;
+    };
+    os << "Total volume: " << slice_stats.totals.volume << std::endl;
+    os << "Distillation volume: " 
+        << slice_stats.totals.distillation_volume 
+        << " (" << get_percentage_volume(slice_stats.totals.distillation_volume) << "%)" << std::endl;
+    os << "Unused routing volume: " 
+        << slice_stats.totals.unused_routing_volume 
+        << " (" << get_percentage_volume(slice_stats.totals.unused_routing_volume) << "%)" << std::endl;
+    os << "Dead volume: "
+        << slice_stats.totals.dead
+        << " (" << get_percentage_volume(slice_stats.totals.dead) << "%)" << std::endl;
+    os << "Other active volume: " 
+        << slice_stats.totals.unused_routing_volume 
```

Comment:
This line is not the output for other active volume.
```suggestion
        << slice_stats.totals.volume - slice_stats.totals.unused_routing_volume -slice_stats.totals.distillation_volume - slice_stats.totals.dead
```

Reply:
Good catch!